.vib/grafana/goss/goss.yaml

```diff
@@ -8,11 +8,6 @@ file:
     exists: true
     mode: "2755"
 command:
-  grafana-cli-data-migration:
-    exec: cd /opt/bitnami/grafana && grafana-cli admin data-migration encrypt-datasource-passwords
-    exit-status: 2
-    stderr:
-      - /duplicate metrics collector registration attempted/
```

Comment:
I'd say we should not remove the test, but rather disable/comment it out (if disabling it is not possible). Additionally, I would also take this opportunity to fix the test, which currently produces a false positive.

Reply:
I changed the test, tell me if that it's okay with you.